javascript/modules/client/src/client-erc20/internal/client/decoding.ts

```diff
@@ -1,17 +1,15 @@
 import { bytesToHex } from "@aragon/sdk-common";
```

Comment:
Same as before, maybe it is OK to have `/client` as an extra folder level... but we already have `client-erc20` itself